File: 백준/Silver/2563. 색종이/색종이.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
    static int N; // 색종이 수
    static Pos[] arr;
    static boolean[][] paper;
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        N = Integer.parseInt(st.nextToken());
        arr = new Pos[N];
        paper = new boolean[100][100];
        for(int i=0; i<N; i++){
            st = new StringTokenizer(br.readLine());
            int x = Integer.parseInt(st.nextToken());
            int y = Integer.parseInt(st.nextToken());
            for(int dx=x; dx<x+10; dx++){
                for(int dy=y; dy<y+10; dy++){
                    paper[dy][dx] = true;
                }
            }
        }
        int answer=0;
        for(int i=0; i<100; i++){
            for(int j=0; j<100; j++){
                if(paper[i][j]) answer++;
            }
        }
        System.out.println(answer);
    }
    static class Pos {
        int x;
        int y;
        public Pos(int x, int y){
            this.x = x;
            this.y = y;
        }
    }
}

File: 백준/Silver/2563. 색종이/색종이.kt
import java.util.*
import java.io.*
fun main()=with(BufferedReader(InputStreamReader(System.`in`))){
    val N = Integer.parseInt(readLine())
    var arr = Array(100){Array(100){0}}
    var r : Int
    var c : Int
    var answer = 0
    for(num in 0 until N){
        val st = StringTokenizer(readLine())
        r = Integer.parseInt(st.nextToken())
        c = Integer.parseInt(st.nextToken())
        for(i in r until r+10){
            for(j in c until c+10){
                if(arr[i][j]==1) continue
                arr[i][j] = 1
                answer++;
            }
        }
    }
    print(answer)
}